### PR TITLE
Adds issue and pull request templates

### LINKED ISSUE #42: Adding templates for PR / Issues and CoC
If this repository gets more attention, should we add a template for

* Issues
* Pull Requests

and a Code of Conduct? For the sake of simplicity, I'd just use either one of:

* https://github.com/stevemao/github-issue-templates
* or the issue template from [elixir](https://github.com/elixir-lang/elixir/blob/master/.github/ISSUE_TEMPLATE.md) with the appropriate modifications

The Code of Conduct can be created [via Github itself](https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project).

### PR DESCRIPTION
**Pull request by floriank**
Fixes #42 

I took the simplest issue and PR templates I could find to just give some structure to the whole thing - I hope it'll give it some structure in time of need* :)

<sub>\* In case @josevalim carpet bombs PRs again, it'll likely slow him down :clown_face:</sub>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (asaaki) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Any notes on the 2 empty lines before the first heading?

Comment (floriank) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Some men just like to see the world 🔥

Comment (floriank) [reply .github/PULL_REQUEST_TEMPLATE.md]: ..or you know... oversight and such and such.

Comment (floriank) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed it 😉

Comment (asaaki) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sorry for my OCD. 💚

**Pull request merged**